jobs/JGLOBAL_FORECAST_FINALIZE

```diff
@@ -0,0 +1,63 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "fcst" -c "base fcst"
+
+##############################################
+# Set variables used in the script
+##############################################
+# Set wave variables
+if [ ${DO_WAVE:-"NO"} = "YES" ]; then
+  # WAVE component directory
+  export CDUMPwave=${CDUMPwave:-${CDUMP}wave}
+  export COMINwave=${COMINwave:-$(compath.py ${envir}/${NET}/${gfs_ver})/${CDUMP}.${PDY}/${cyc}/wave}
+  export COMOUTwave=${COMOUTwave:-$(compath.py -o ${NET}/${gfs_ver})/${CDUMP}.${PDY}/${cyc}/wave}
+fi
+
+##############################################
+# Begin JOB SPECIFIC work
+##############################################
+
+# Restart conditions for GFS cycle come from GDAS
+rCDUMP=${CDUMP}
+[[ ${CDUMP} = "gfs" ]] && export rCDUMP="gdas"
```

Comment:
@WalterKolczynski-NOAA I just made a copy of file had in another branch. I am guess my rebase didn't take. Before any development/refactoring proceeds I will make sure I begin from a branch UTD with `develop`. Thank you for the heads-up.

Reply:
This is in develop too. Trying not to repeat its mistakes. `CDUMP` can only be `gdas` or `gfs` (`enkfgdas` and `enkfgfs` have a different j-job for now). Changing `gfs` to `gdas` just means it is always `gdas`, so let's just say that. (If the enkf script gets combined into this one, I think it would instead be `rCDUMP=${CDUMP/gfs/gdas}`.)